library/network/src/modules/SuSEFirewall.rb

```diff
@@ -909,6 +909,61 @@ def AddAllowedPortsOrServices(add_ports, protocol, zone)
       nil
     end
 
+    # Function returns if any other firewall then SuSEfirewall2 is currently running on the
```

Comment:
Now that the method is moved to the base class, which firewalls are considered "other" firewalls? Other than SF2? Other than SF2 or FWD? Other than the one currently running?